<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchInfoStudent</class>
 <widget class="QWidget" name="searchInfoStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>761</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>130</y>
     <width>201</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>提交信息修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>撤销修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>添加学生信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>删除选中行</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>为选中学生添加课程</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_12">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>更新/查看当前学生选课</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_13">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>更新/查看学生绩点</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_search_number"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_search_name"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>更新/显示所有学生信息</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_10">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>NAME:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_11">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>580</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_14">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>学号升序排列</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_15">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>GPA降序排列</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>590</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_depart"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>查看某系学生成绩</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>727</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>tableView</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
